go.mod

```diff
@@ -255,5 +256,5 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.44.2-celestia-rc-1
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/libp2p/go-libp2p-pubsub v0.5.7-0.20211029175501-5c90105738cf => github.com/celestiaorg/go-libp2p-pubsub v0.5.6-celestia
-	github.com/tendermint/tendermint v0.34.14 => github.com/celestiaorg/celestia-core v0.34.15-celestia
+	github.com/tendermint/tendermint v0.34.14 => github.com/celestiaorg/celestia-core v0.34.16-celestia-rc-1
```

Comment:
Is there a particular reason we aren't using the most recent release here? https://github.com/celestiaorg/celestia-core/releases/tag/v1.1.0-tm-v0.34.16

Reply:
This release was added after merging additional nmt method. Will update to [v1.1.0-tm-v0.34.16](https://github.com/celestiaorg/celestia-core/releases/tag/v1.1.0-tm-v0.34.16)

Reply:
Thanks! I'm surprised that go.sum is different though go.mod stayed the same: https://github.com/celestiaorg/celestia-node/pull/545/files#r861699632